R-package/R/lgb.model.dt.tree.R

```diff
@@ -90,14 +90,25 @@ lgb.model.dt.tree <- function(model, num_iteration = NULL) {
 
 #' @importFrom data.table := data.table rbindlist
 .single_tree_parse <- function(lgb_tree) {
+  long_list <- c(
```

Comment:
```suggestion
  tree_info_cols <- c(
```

Totally support pulling this up into its own vector and doing it here so it's no re-created on every `pre_order_traversal()` call.

But could you please use a more informative name?